source/deprecations/ember-data/v2.x.html.md

```diff
@@ -676,3 +676,36 @@ Using `JSONAPISerializer.payloadKeyFromModelName` to serialize the type of a
 See [JSONAPISerializer.payloadKeyFromModelName for
 Resource](#toc_jsonapiserializer-payloadkeyfrommodelname-for-resource) for more
 information.
+
+#### Unused Initializers
+
+###### until: 3.0.0
+###### id: ds.deprecated-initializers
+
+The Ember Data initializers `data-adapter`, `injectStore`, `transforms`, and `store` are no longer used, so they are being removed. Apps that depend on these for the ordering of their own custom initializers can substitute `ember-data` instead, without any change in functionality.
```

Comment:
Can you replace "Apps" with "Applications" and add a newline after the "removed." full stop?

Like so:

```
… removed.
Applications that …
```